src/AbstractServer.php

```diff
@@ -111,29 +100,28 @@ protected function _buildSignature(Reflection\AbstractFunction $reflection, $cla
     /**
      * Dispatch method
      *
-     * @deprecated Since 2.7.0; method will be renamed to remove underscore
-     *     prefix in 3.0.
-     *
      * @return mixed
      * @throws ReflectionException
      */
-    // @codingStandardsIgnoreStart
-    protected function _dispatch(Method\Definition $invokable, array $params)
+    protected function dispatch(Method\Definition $invokable, array $params)
     {
-    // @codingStandardsIgnoreEnd
         $callback = $invokable->getCallback();
         $type     = $callback->getType();
 
         if ('function' === $type) {
             $function = $callback->getFunction();
+            // phpcs:disable
             return call_user_func_array($function, $params);
+            // phpcs:enable
         }
 
         $class  = $callback->getClass();
         $method = $callback->getMethod();
 
         if ('static' === $type) {
+            // phpcs:disable
             return call_user_func_array([$class, $method], $params);
+            // phpcs:enable
```

Comment:
```suggestion
            $callback = [$class, $method];
            return $callback(...$params);
```

Same comment here; simplify! Doing so removes the need for ignoring CS rules.

Reply:
Introduce callable separately?